azkaban-exec-server/src/main/java/azkaban/execapp/FlowPreparer.java

```diff
@@ -158,10 +179,27 @@ void setup(final ExecutableFlow flow) throws ExecutorManagerException {
       FileIOUtils.deleteDirectorySilently(tempDir);
       LOGGER.error("Error in preparing flow execution {}", flow.getExecutionId(), ex);
       throw new ExecutorManagerException(ex);
-    } finally {
-      if (projectFileHandler != null) {
-        projectFileHandler.deleteLocalFile();
-      }
+    }
+  }
+
+  public void setupContainerizedExecution(final ExecutableFlow flow, final File projectDir)
+    throws ExecutorManagerException {
+    final ProjectDirectoryMetadata projectDirMetadata = new ProjectDirectoryMetadata(
+              flow.getProjectId(),
+              flow.getVersion());
+    final long flowPrepStartTime = System.currentTimeMillis();
+    final int execId = flow.getExecutionId();
+
+    LOGGER.info("Deepak : Using containerized execution flow setup");
```

Comment:
Please remove your name from log line. Remove the log line if not needed.

Reply:
Oops